mobile-workspace/latest/using/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Using Mobile Workspace
+---
+
+## Searching
+
+* Search is persistent and the Mobile Workspace allows global searching at any time. The results can be quickly filtered by files, folders or libraries just like in the Digital Workspace.
+* The Mobile Workspace keeps a recent history of search terms so that previous searches can be quickly repeated.
+* Contextual search is available within folders. This search mode will only search within that folder but can be converted to a global search using the quick filter. You can also filter by files and folders within the contextual folder search.
+
+## Recent Files
+
+The Recent Files view shows all the files that have been created or modified within the last 30 days by the logged in user. The Search API is used to query SOLR for changes made by the user and includes an extra column to display where the file is located in the content repository.
```

Comment:
Please drop SOLR as there may alternative search providers used. Maybe rewrite it as:
The Search API is used to query for changes made by the user [...]

Reply:
done